core/toolbox/toolbox.js

```diff
@@ -392,12 +392,21 @@ Blockly.Toolbox.prototype.renderContents_ = function(toolboxDef) {
  * @private
  */
 Blockly.Toolbox.prototype.renderToolboxItem_ = function(toolboxItemDef, fragment) {
+  var kind = toolboxItemDef['kind'];
+
+  if (kind.toUpperCase() == 'CATEGORY' &&
```

Comment:
Add a comment explaining this extra conversion.

Reply:
done